src/grandorgue/GOFrame.cpp

```diff
@@ -222,7 +222,7 @@ GOFrame::GOFrame(
     wxITEM_NORMAL);
   m_file_menu->AppendSeparator();
   m_file_menu->Append(
-    ID_SETTINGS, wxT("&Settings..."), wxEmptyString, wxITEM_NORMAL);
+    ID_SETTINGS, _("&Settings..."), wxEmptyString, wxITEM_NORMAL);
```

Comment:
`Settings` was not translated by design: it allows to user find this menu item whenever the corrent language is wrong.